scripts/plot_network.py

```diff
@@ -264,10 +264,18 @@ def plot_h2_map(network):
         axis=1
     )
 
-    h2_retro = h2_retro["p_nom_opt"]
+    h2_new.index = h2_new.apply(
+        lambda x: f"H2 pipeline {x.bus0.replace(' H2', '')} -> {x.bus1.replace(' H2', '')}"
+    ,axis=1)
 
+    h2_new = h2_new["p_nom_opt"].groupby(level=0).sum()
+    h2_retro = h2_retro["p_nom_opt"].groupby(level=0).sum()
+    h2_retro = h2_retro.groupby(level=0).sum().reindex(h2_new.index).fillna(0)
+
+    n.links.rename(index=lambda x: x.split("-2")[0], inplace=True)
+    n.links = n.links.groupby(level=0).first()
```

Comment:
Not sure I 100% understood what's going on here.

Reply:
I tried to make it clearer now